rcl_interfaces/msg/ParameterDescriptor.msg

```diff
@@ -2,5 +2,7 @@
 
 string name
 uint8 type # Enum defined in ParameterType.msg
+# If 'false' then the value cannot change after it has been initialized
+bool mutable_after_initialization true
```

Comment:
Do you think that `read_only` could be a descriptive enough name ? (and defaulting to `false` if we change the name)

Reply:
I think so. I think it's possible someone might think `read_only` means a run-time user can't initialize it with a value, or they might think a run-time user can't change the value but the node itself might change it and report changes on the `parameter_events` topic. However, the documentation can explain that here.

`dynamic` defaulting to `true` is another option, but that term relies on associating it with `dynamic_reconfigure` in ROS 1.

Reply:
changed to `read_only` in dab17415e19351a117cce3c69af67540f93bb9ba